src/auto/settings.rs

```diff
@@ -139,240 +177,217 @@ pub struct SettingsBuilder {
+            }
+if let Some(ref enable_developer_extras) = self.enable_developer_extras {
+                properties.push(("enable-developer-extras", enable_developer_extras));
+            }
+if let Some(ref enable_dns_prefetching) = self.enable_dns_prefetching {
+                properties.push(("enable-dns-prefetching", enable_dns_prefetching));
+            }
         #[cfg(any(feature = "v2_20", feature = "dox"))]
-        {
-            if let Some(ref enable_encrypted_media) = self.enable_encrypted_media {
+if let Some(ref enable_encrypted_media) = self.enable_encrypted_media {
                 properties.push(("enable-encrypted-media", enable_encrypted_media));
             }
-        }
-        if let Some(ref enable_frame_flattening) = self.enable_frame_flattening {
-            properties.push(("enable-frame-flattening", enable_frame_flattening));
-        }
-        if let Some(ref enable_fullscreen) = self.enable_fullscreen {
-            properties.push(("enable-fullscreen", enable_fullscreen));
-        }
-        if let Some(ref enable_html5_database) = self.enable_html5_database {
-            properties.push(("enable-html5-database", enable_html5_database));
-        }
-        if let Some(ref enable_html5_local_storage) = self.enable_html5_local_storage {
-            properties.push(("enable-html5-local-storage", enable_html5_local_storage));
-        }
-        if let Some(ref enable_hyperlink_auditing) = self.enable_hyperlink_auditing {
-            properties.push(("enable-hyperlink-auditing", enable_hyperlink_auditing));
-        }
-        if let Some(ref enable_java) = self.enable_java {
-            properties.push(("enable-java", enable_java));
-        }
-        if let Some(ref enable_javascript) = self.enable_javascript {
-            properties.push(("enable-javascript", enable_javascript));
-        }
+if let Some(ref enable_frame_flattening) = self.enable_frame_flattening {
+                properties.push(("enable-frame-flattening", enable_frame_flattening));
+            }
+if let Some(ref enable_fullscreen) = self.enable_fullscreen {
+                properties.push(("enable-fullscreen", enable_fullscreen));
+            }
+if let Some(ref enable_html5_database) = self.enable_html5_database {
+                properties.push(("enable-html5-database", enable_html5_database));
+            }
+if let Some(ref enable_html5_local_storage) = self.enable_html5_local_storage {
+                properties.push(("enable-html5-local-storage", enable_html5_local_storage));
+            }
+if let Some(ref enable_hyperlink_auditing) = self.enable_hyperlink_auditing {
+                properties.push(("enable-hyperlink-auditing", enable_hyperlink_auditing));
+            }
+if let Some(ref enable_java) = self.enable_java {
+                properties.push(("enable-java", enable_java));
+            }
+if let Some(ref enable_javascript) = self.enable_javascript {
```

Comment:
I don't know if it only has the wrong indent in GitHub, but this indent seems off.

Reply:
No, indentation is weird here. But then again this is auto-generated code that should not be modified.

Reply:
Agreed. That's a bug in gir, though.